src/modules/robot/arm_solutions/RotatableDeltaSolution.cpp

```diff
@@ -0,0 +1,253 @@
+#include "RotatableDeltaSolution.h"
+#include "checksumm.h"
+#include "ConfigValue.h"
+#include "ConfigCache.h"
+#include "libs/Kernel.h"
+#include "libs/nuts_bolts.h"
+#include "libs/Config.h"
+#include "libs/utils.h"
+#include "StreamOutputPool.h"
+#include <fastmath.h>
+
+#define delta_e_checksum       			CHECKSUM("delta_e_checksum")
+#define delta_f_checksum   	    		CHECKSUM("delta_f_checksum")
+#define delta_re_checksum       		CHECKSUM("delta_re_checksum")
+#define delta_rf_checksum       		CHECKSUM("delta_rf_checksum")
+#define delta_z_offset_checksum 		CHECKSUM("delta_z_offset_checksum")
+
+#define delta_ee_offs_checksum 		  	CHECKSUM("delta_ee_offs_checksum")
+#define tool_offset_checksum			CHECKSUM("tool_offset_checksum")
+
+#define z_home_angle_checksum        		CHECKSUM("z_home_angle_checksum")
+
+#define delta_printable_radius_checksum    	CHECKSUM("delta_printable_radius_checksum")
+
+#ifndef alpha_steps_per_mm_checksum
```

Comment:
Hi Jim,

Good point.

I will remove all the code intended for future feature sets and make it a clean implementation and later create a new branch for the next feature set.

It will be the final code for a Rotational Delta arm solution, munis future to-be-implemented features.

Cheers
Douglas
> On 11/06/2015, at 12:10 pm, Jim Morris <notifications@github.com> wrote:
> 
> In src/modules/robot/arm_solutions/RotatableDeltaSolution.cpp <https://github.com/Smoothieware/Smoothieware/pull/673#discussion_r32179776>:
> 
> > +#include <fastmath.h>
> > +
> > +#define delta_e_checksum       			CHECKSUM("delta_e_checksum")
> > +#define delta_f_checksum   	    		CHECKSUM("delta_f_checksum")
> > +#define delta_re_checksum       		CHECKSUM("delta_re_checksum")
> > +#define delta_rf_checksum       		CHECKSUM("delta_rf_checksum")
> > +#define delta_z_offset_checksum 		CHECKSUM("delta_z_offset_checksum")
> > +
> > +#define delta_ee_offs_checksum 		  	CHECKSUM("delta_ee_offs_checksum")
> > +#define tool_offset_checksum			CHECKSUM("tool_offset_checksum")
> > +
> > +#define z_home_angle_checksum        		CHECKSUM("z_home_angle_checksum")
> > +
> > +#define delta_printable_radius_checksum    	CHECKSUM("delta_printable_radius_checksum")
> > +
> > +#ifndef alpha_steps_per_mm_checksum
> well if this is not the final version then please hold off pushing it until it is final. Just let me know when you are done. I'm sorry to be pushy but we have a very high standard of code review, and it is very time consuming to review every pull request, hence I really do not want to do this piece meal. Thank you On 06/10/2015 05:06 PM, Douglas Pearless wrote In src/modules/robot/arm_solutions/RotatableDeltaSolution.cpp
> … <x-msg://123/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/Smoothieware/Smoothieware/pull/673#discussion_r32179603> <https://github.com/Smoothieware/Smoothieware/pull/673#discussion_r32177039> <https://github.com/Smoothieware/Smoothieware/pull/673/files#r32177039> <https://github.com/Smoothieware/Smoothieware/pull/673/files#r32179603>
> —
> Reply to this email directly or view it on GitHub <https://github.com/Smoothieware/Smoothieware/pull/673/files#r32179776>.
>